aura-borealis-flask-app/app.py

```diff
@@ -520,13 +473,21 @@ def loadData(name):
 		cacheData(init_all_unique_warnings, 'init_all_unique_warnings')
 		cacheData(init_all_severities, 'init_all_severities')
 		cacheData(init_all_percentiles, 'init_all_percentiles')
-		print('init_all_percentiles', init_all_percentiles)'''
+		print('init_all_percentiles', init_all_percentiles)
+
+		all_warning_types = get_unique_warning_types()
```

Comment:
@kdobolyi, could you please add a brief comment at the top of this code chunk explaining what is going on?

Reply:
done!